setup.py

```diff
@@ -9,7 +9,14 @@
     author="Mairie de Paris, Incubateur de Services Numériques (SGMAP)",
     packages=find_packages(),
     include_package_data=True,
-    install_requires=[],
+    install_requires=[
+        'OpenFisca-France >= 17.0, < 18.0'
+    ],
+    extras_require = {
```

Comment:
Unnecessary, this installs `nose` and `flake8`, not need by `openfisca-run-test`.